edge/scheduler/logs_test.go

```diff
@@ -12,7 +12,7 @@ func TestDataRace(t *testing.T) {
 	cli := client.NewPortainerClient(
 		"portainerURL",
 		"endpointID",
-		"edgeID",
+		func() string { return "edgeID" },
```

Comment:
doesn't really matter, but probably should be (include the line above)
```suggestion
		"edgeID"
		func() string { return "endpointID" },
```